scripts/aggregate-translations/aggregate-translations-cli.js

```diff
@@ -6,11 +6,17 @@ const parseCLIList = require('../utils/parse-cli-list');
 
 const aggregateTranslations = require('./aggregate-translations');
 
+const convertDirectoryPath = directoryPath => directoryPath.split('/').join(path.sep);
```

Comment:
@dev-cprice, Would you mind dropping the `convertDirectoryPath` function. We are using glob to search for the paths and glob's documentation state to only use forward slashes in the patterns: https://www.npmjs.com/package/glob#windows.

I didn't realize/know this when I originally implemented this concept.